packages/eslint-config-airbnb/rules/style.js

```diff
@@ -34,6 +34,8 @@ module.exports = {
     'lines-around-comment': 0,
     // disallow mixed 'LF' and 'CRLF' as linebreaks
     'linebreak-style': 0,
+    // specify the maximum length of a line in your program
+    "max-len": [2, 100, 4],
```

Comment:
re the third option: since we use 2-space indentation, we should probably count this as `2` (even though we don't use tabs).

Also, can we use single quotes, and the object form, and include all defaults? For example:
```
'max-len': [2, {
  "code": 100,
  "comments": 100,
  "commentLength": 100,
  "tabWidth": 2,
  "ignoreUrls": false,
  "ignorePattern": null,
  "ignoreTrailingComments": false,
  "ignoreComments": false
}],
```